slave.mk

```diff
@@ -581,6 +581,7 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
 	export swsssdk_py2_wheel_path="$(addprefix $(PYTHON_WHEELS_PATH)/,$(SWSSSDK_PY2))"
 	export platform_common_py2_wheel_path="$(addprefix $(PYTHON_WHEELS_PATH)/,$(SONIC_PLATFORM_COMMON_PY2))"
 	export redis_dump_load_py2_wheel_path="$(addprefix $(PYTHON_WHEELS_PATH)/,$(REDIS_DUMP_LOAD_PY2))"
+	export install_debug_image="$(INSTALL_DEBUG_TOOLS)"
```

Comment:
Since `INSTALL_DEBUG_TOOLS` now does more than just install extra debugging tools, I think we could find more appropriate names for these variables, like maybe `BUILD_DEBUG_IMAGE` or simply `DEBUG_IMAGE`

Reply:
I am in sync with you. A revisit of this new work (debug docker FW) and some refining is in order. If you have cycles, can you please help review the work in master? 
The cleanup can be preferably done in master.
I prefer to restrict changes to Nov image, unless it is *required*.

Reply:
Agreed.

>If you have cycles, can you please help review the work in master?

Is there a PR? I didn't see one.

Reply:
There could be a couple. I will get them to you.